src/components/AppstoreButton/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { ReactElement } from 'react'
+import AppstoreLightBadge from 'src/assets/icons/appstore-light.png'
+import AppstoreDarkBadge from 'src/assets/icons/appstore-dark.svg'
+
+enum LINKS {
+  footer = 'https://apps.apple.com/app/apple-store/id1515759131?pt=119497694&ct=Web%20App%20Footer&mt=8',
```

Comment:
I would ask @sche if we wants to create a campaign for this.

Reply:
That's who I got the link form